### PR TITLE
Add example of how to crash from a pinvoke

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: main/build/MacOSX/monostub.mm [hunk 1]
```diff
@@ -227,6 +227,12 @@
 	}
 }
 
+extern "C" void crash_me()
```
Comment (Therzok) [inline main/build/MacOSX/monostub.mm]: Need to figure out how to include this only in debug builds.

Comment (Therzok) [reply main/build/MacOSX/monostub.mm]: Adding do not merge until this is fixed.

File: main/src/addins/PerformanceDiagnostics/PerformanceDiagnostics/Commands.cs [hunk 2]
```diff
@@ -201,5 +201,16 @@ protected override void Run ()
...
 	}
+
+	class InduceNativePInvokeCrashHandler : CommandHandler
+	{
+		[DllImport ("__Internal")]
+		static extern void debug_trigger_sigsegv ();
+
+		protected override void Run ()
+		{
+			crash_me ();
```
Comment (kdubau) [inline main/src/addins/PerformanceDiagnostics/PerformanceDiagnostics/Commands.cs]: Should be `debug_trigger_sigsegv ();` ?

Comment (Therzok) [reply main/src/addins/PerformanceDiagnostics/PerformanceDiagnostics/Commands.cs]: Yup!

Comment (Therzok) [inline main/src/addins/PerformanceDiagnostics/PerformanceDiagnostics/Commands.cs]:
```suggestion
			debug_trigger_sigsegv ();
```

**Pull request merged**